src/signals/IncidentMap/components/Header/styled.tsx

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import { breakpoint, themeColor, themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+import { HEADER_HEIGHT_DESKTOP, HEADER_HEIGHT_MOBILE } from './constants'
+
+export const HeaderWrapper = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  height: ${themeSpacing(HEADER_HEIGHT_DESKTOP)};
+  width: 100%;
+  padding: 0 ${themeSpacing(4)};
+  background-color: ${themeColor('tint', 'level1')};
+  box-shadow: 0 ${themeSpacing(1)} ${themeSpacing(1)} 0 rgba(0, 0, 0, 0.1);
+  z-index: 2;
```

Comment:
why is this z index 2? maybe a good idea for this repo to make a config file with named z indexes. That way you always have a reference to this somewhat magic number.

Reply:
sharp. This was because the header and footer of the regular application were still rendered. I removed them when on /meldingenkaart and set the index of the map to be `0` and the header relative to that.